build.sh

```
@@ -2,122 +2,34 @@
 
 set -ouex pipefail
 
-RELEASE="$(rpm -E %fedora)"
+export RELEASE="$(rpm -E %fedora)"
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)